tests/security/test_tls.py

```diff
@@ -0,0 +1,46 @@
+import pytest
+import ssl
+import socket
+import logging
+
+# This test is designed to verify that TLS connections is secured
+#
+# Requirements:
+# - An XCP-ng host
+
+@pytest.mark.parametrize("protocol", [ssl.PROTOCOL_TLSv1, ssl.PROTOCOL_TLSv1_1])
+def test_tls_disabled(host: str, protocol):
```

Comment:
those constants are numeric, that makes both test naming and logging hard to understand:

```
tests/security/test_tls.py::test_tls_disabled[3] 
...
May 28 13:34:11.502 INFO Testing if protocol 3 is disabled on host 10.1.15.40
```